### DISCUSSION TITLE
Hotkey to change the name of selected track

### DISCUSSION THREAD

**Discussion by mapan001** [audacity/audacity#7107] [Far future]
Hello everyone,
I have been using Audacity for many years to record church services. With the last update, the button to open the dropdown menu of an audio track became very small. 
My recording sessions usually consist of 15 or more tracks. For a better overview and orientation, I give each track a name. 
At this point, I miss a hotkey to open the window for changing the name of the selected track. 

Would it be a big effort to add this little helper?

**Comment by petersampsonaudacity** [comment]
@mapan001 

You can also get the Track Control Panel's dropdown menu by right-clicking anywhere in the Track Control Panel,

**Comment by DavidBailes** [comment]
> At this point, I miss a hotkey to open the window for changing the name of the selected track.

The shortcut for opening the focused track's menu is either the application key (which is used in general for opening context menus), or Shift + M. And the Rename command on the menu has the access key N.
So to open the Set track name dialog, you could press either:
-  Application key, then N, or
-  Shift + M, then N.
